package-lock.json

```diff
@@ -0,0 +1,3997 @@
+{
```

Comment:
Can't comment on it directly so I'll leave a comment here: the `.DS_store` file is your local metadata and we don't want that in the repository. Please remove it from the PR.